src/llmq/signing.cpp

```diff
@@ -633,12 +633,7 @@ void CSigningManager::ProcessRecoveredSig(const std::shared_ptr<const CRecovered
     WITH_LOCK(cs_pending, pendingReconstructedRecoveredSigs.erase(recoveredSig->GetHash()));
 
     if (m_mn_activeman != nullptr) {
-        CInv inv(MSG_QUORUM_RECOVERED_SIG, recoveredSig->GetHash());
-        connman.ForEachNode([&](const CNode* pnode) {
```

Comment:
oh this change is soo good! Look to this one: 6149288d4818b7e5700657755104b95cbb03306c

Maybe you can drop all usages of CConnman from llmq code?

Reply:
Aye! Awesome, didn't even realize I'd removed all usages

Reply:
@knst see: https://github.com/dashpay/dash/pull/6425 one more bites the dust!